src/index.css

```
@@ -1,19 +1,24 @@
 
 @theme {
-  --color-primary: #0ea5e9;
+  --color-primary: #2046ff;
   --color-primary-foreground: #f8fafc;
 
-  --color-bg: #020617;
-  --color-surface: #0f172a;
-  --color-text: #e2e8f0;
-  --color-muted: #94a3b8;
+  --font-sans:
+    "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.7.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)